src/login/login.js

```diff
@@ -15,6 +15,20 @@ export default class Login extends Component {
     this.state = { username: '', password: '' };
   }
 
+  validateUser() {
+    this.props.socket.emit('validateUserLogin', { 
+        username: this.state.username,
+        password: this.state.password
+    });
+  }
+
+  componentWillMount() {
+    this.props.socket.on('Authentication', username => {
+      console.log('heyo2');
```

Comment:
Check the 'log' function and wrapper on Maps.js to get around eslint issues with console.log